pyV2DL3/vegas/VegasDataSource.py

```diff
@@ -42,16 +45,22 @@ def __del__(self):
         """
         cpy_nonestring = "<class 'CPyCppyy_NoneType'>"
 
-        if str(type(self.__evt_file__)) != cpy_nonestring and not isinstance(self.__evt_file__, str):
+        if str(type(self.__evt_file__)) != cpy_nonestring and not isinstance(
+            self.__evt_file__, str
+        ):
             self.__evt_file__.closeTheRootFile()
 
     def __fill_evt__(self):
-        gti, ea_config, evt_dicts = __fillEVENTS_not_safe__(self.__evt_file__, self.__ea_files__,
-                                                            event_class_mode=self.__event_class_mode__,
-                                                            fov_cut=self.__fov_cut__,
-                                                            reco_type=self.__reco_type__,
-                                                            save_msw_msl=self.__save_msw_msl__,
-                                                            )
+        gti, ea_config, evt_dicts = __fillEVENTS_not_safe__(
+            self.__evt_file__,
+            self.__ea_files__,
+            self.__irf_to_store__,
+            event_class_mode=self.__event_class_mode__,
+            fov_cut=self.__fov_cut__,
+            reco_type=self.__reco_type__,
+            save_msw_msl=self.__save_msw_msl__,
+            psf_king_params=self.__psf_king_params__,
```

Comment:
This method now also needs `corr_EB` parameter so that it can used. Overall it needs some reworking to put with the right strategy.

Reply:
[7382c68](https://github.com/VERITAS-Observatory/V2DL3/pull/198/commits/7382c68606053482a186eb8cd1c39105dc325a35) 

Added command line prompt